recipe/meta.yaml

```diff
@@ -1,61 +1,103 @@
-{% set name = "onnx" %}
 {% set version = "1.15.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: onnx-split
   version: {{ version }}
 
 source:
   url: https://github.com/onnx/onnx/archive/v{{ version }}.tar.gz
   sha256: c757132e018dd0dd171499ef74fca88b74c5430a20781ec53da19eb7f937ef68
   patches:
-    - 0001-Link-to-abseil_dll.patch  # [win and (libprotobuf != "3.21")]
+    - patches/0001-Link-to-abseil_dll.patch
+    - patches/0002-Let-conda-forge-define-CXX-standard.patch
+    - patches/0003-include-cinttypes-in-cpp2py_export.cc.patch
+    - patches/0004-Use-new-FindPython.patch
+    - patches/0005-Link-protobuf-privately.patch
+    - patches/0006-Install-shared-windows-libraries.patch
 
 build:
-  number: 3
-  entry_points:
-    - check-model = onnx.bin.checker:check_model
-    - check-node = onnx.bin.checker:check_node
-    - backend-test-tools = onnx.backend.test.cmd_tools:main
+  number: 4
 
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - cmake
-    - make
-    - libprotobuf
-  host:
-    - python
-    - pip
-    - protobuf
-    - libprotobuf
-    - pytest-runner
-    - ninja
-    - pybind11
-    - numpy
-  run:
-    - python
-    - protobuf
-    - {{ pin_compatible('numpy') }}
-    - typing-extensions >=3.6.2.1
-
-test:
-  requires:
-    - pip
-  imports:
-    - onnx
-  commands:
-    - pip check
-    - check-model --help
-    - check-node --help
-    - backend-test-tools --help
-    - test -f "$PREFIX/lib/libonnx${SHLIB_EXT}"  # [unix]
-    - test -f "$PREFIX/lib/cmake/ONNX/ONNXConfig.cmake"  # [unix]
+outputs:
+  - name: libonnx
```

Comment:
Can't remember ever seeing you voluntarily turn something into a multi-output recipe 😜 

But looks like a nice improvement to me in general 💪

Reply:
oh troubleshooting is terrible. don't get me wrong.....